docs/en/observability/monitor-status-alert.asciidoc

```diff
@@ -4,6 +4,12 @@
 <titleabbrev>Monitor status</titleabbrev>
 ++++
 
+//REVIEWERS:  I haven't updated this topic because I'm not sure if this page should exist.
+//I don't see the Uptime app in the UI anymore, but maybe it's only available if users have
+//installed Heartbeat? If so, users who see this topic may get confused. Should I remove this topic?
+//Looks like the alerting settings for synthetics are covered here: https://www.elastic.co/guide/en/observability/8.13/synthetics-settings.html#synthetics-settings-alerting
+//Also looks like the UI under Settings only covers a subset of the connectors that are available for other types of observability rules. Why?
```

Comment:
> By default, Uptime is no longer shown if they have no Heartbeat data, because the preferred approach is to use the Synthetics Kibana app.  Synthetics provides a better experience for managing monitors than Uptime (both through the [UI](https://www.elastic.co/guide/en/observability/current/synthetics-get-started-ui.html), and/or as [project monitors](https://www.elastic.co/guide/en/observability/current/synthetics-get-started-project.html).

Reply:
> All new users  have it turned off by default but it's still there to turn on. Existing users will continue to see Uptime.